lib/ansible/modules/remote_management/dellemc/idrac/dellemc_idrac_firmware.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# _*_ coding: utf-8 _*_
```

Comment:
It's an interesting one (never saw such style). Sure this probably works, but the common format is
```suggestion
# -*- coding: utf-8 -*-
```